apps/judicial-system/web/src/routes/Shared/Requests/Requests.tsx

```diff
@@ -35,6 +36,8 @@ export const Requests: React.FC = () => {
   const isProsecutor = user?.role === UserRole.PROSECUTOR
   const isJudge = user?.role === UserRole.JUDGE
   const isRegistrar = user?.role === UserRole.REGISTRAR
+  const isHighCourtJudge =
```

Comment:
A better name for this variable would probably be isHighCourtUser as the user could either be a Judge or a Registrar.

Reply:
Thank you, fixed